### ISSUE TITLE
In reply to...

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#29]
```
More often than not I want to see in a reply to which exactly message it
is. A "In reply to..." link to the original message would be useful.
```

Original issue reported on code.google.com by `isagal...@gmail.com` on 3 Jan 2010 at 1:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 3 Jan 2010 at 2:39
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by ffee027b6b revision.
```

Original comment by `tro...@gmail.com` on 6 Jan 2010 at 3:25
- Changed state: **Fixed**
